src/components/molecules/InputWrapper/InputWrapper.tsx

```diff
@@ -29,14 +30,20 @@ const InputWrapper = forwardRef<HTMLInputElement, InputWrapperProps>(
       wrapperClass,
       errorClass,
       errorZIndex,
+      paginationLabelClassName,
```

Comment:
**[bloker]**

Do we need this ? We already have the option to pass "className" prop. If you want to overwrite any styles from the "classes.container" then you can do so inside the class passed as "className". It might be slightly more css for this 1 case, but if we start adding extra props to a commonly used componend for every usecase, we will end up with something that will be quite unmaintainable in the future. I would pick maintainability + readability over less code in this case.